scripts/validate_schemas.js

```diff
@@ -0,0 +1,73 @@
+// @ts-check
+
+// Grab the schema for reaction out of the node_modules
+// to see if it is a subset of the production schema
+//
+// Used both by Danger during the deploy PR, and also
+// before the deployment on circle
+//
+const { readFileSync } = require("fs")
+const {
+  introspectionQuery,
+  buildClientSchema,
+  printSchema,
+  buildSchema,
+  findBreakingChanges,
```

Comment:
so there's first class support for this in `graphql-js`? Very cool!

Reply:
Yeah, I was surprised too!